src/components/Camera/index.js

```diff
@@ -50,14 +47,14 @@ const reload = () => window.location.reload()
 // support the precise resolution.
 
 export const CameraPure = ({method, title, subTitle, onUploadFallback,
-                            onFallbackClick, hasError, onUserMedia,
+                            onFallbackClick, hasCameraError, onUserMedia,
                             onFailure, webcamRef, isFullScreen, i18n,
-                            video}: CameraPureType) => (
-  <div className={style.camera}>
-    <Title {...{title, subTitle, isFullScreen}} smaller={true}/>
```

Comment:
Why are you not passing `isFullScreen` anymore? I think we still need this prop in the `Title` component.

Reply:
Thanks @stefaniacardenas, I accidentally removed it while fixing conflicts.